aleph_message/models/execution/volume.py

```diff
@@ -76,3 +76,15 @@ def is_read_only(self):
 
 
 MachineVolume = Union[ImmutableVolume, EphemeralVolume, PersistentVolume]
+
+
+def parse_volume(volume_dict: Union[Dict, MachineVolume]) -> MachineVolume:
+    if isinstance(volume_dict, MachineVolume):
+        return volume_dict
+    for volume_type in [ImmutableVolume, PersistentVolume, EphemeralVolume]:
+        try:
+            return volume_type.parse_obj(volume_dict)
+        except ValueError:
+            continue
+    else:
+        raise ValueError(f"Could not parse volume: {volume_dict}")
```

Comment:
Can we forward the error using something like this ?
```suggestion
def parse_volume(volume_dict: Union[Dict, MachineVolume]) -> MachineVolume:
    if isinstance(volume_dict, MachineVolume):
        return volume_dict
    
    last_error = None
    for volume_type in [ImmutableVolume, PersistentVolume, EphemeralVolume]:
        try:
            return volume_type.parse_obj(volume_dict)
        except ValueError as error:
            last_error = error
            continue
    else:
        raise ValueError(f"Could not parse volume: {volume_dict}") from last_error
```